Modify AppId editing example to match starter pack

martinbherbener opened the pull request:
Downloadable starter pack indicates that when configuring local accounts the IdentityExperienceFrameworkAppId value should be used as IdTokenAudience (and resourceID) within the login-NonInteractive section of TrustFrameworkExtensions; however, the example on this page could be read to indicate that the IdentityExperienceFrameworkAppId  value should be used for client_id, causing confusion.

azurecla commented:
Hi __@rcarmo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

deneha commented:
@rcarmo This pull request has been open for over a month.  We cannot accept your contribution until the licensing agreement request has been addressed.  We'll give you a bit more time to finish up.  If we don't hear back soon, we'll close this PR.

martinbherbener committed:
Modify AppId editing example to match starter pack

Downloadable starter pack indicates that when configuring local accounts the IdentityExperienceFrameworkAppId value should be used as IdTokenAudience (and resourceID) within the login-NonInteractive section of TrustFrameworkExtensions; however, the example on this page could be read to indicate that the IdentityExperienceFrameworkAppId  value should be used for client_id, causing confusion.
